.buildkite/prepare-bidding.yml

```diff
@@ -0,0 +1,83 @@
+agents:
+  queue: "snapshots"
+
+env:
+  gs_bucket: gs://marinade-validator-bonds-mainnet
+  claim_type: "bidding"
+
+steps:
+  - label: ":hammer_and_wrench: :rust: Build"
+    commands:
+    - '. "$HOME/.cargo/env"'
+    - 'cargo build --release --bin bid-distribution-cli'
+    artifact_paths:
+      - target/release/bid-distribution-cli
+
+  - input: "What is the epoch to generate settlements for?"
+    fields:
+      - text: "Epoch"
+        key: "epoch"
+        format: "[0-9]+"
+    if: "build.env('EPOCH') == null"
+  - command: "echo 'Epoch is: $EPOCH'"
+    if: "build.env('EPOCH') != null"
+
+  - wait: ~
+
+  - label: ":closed_lock_with_key: Concurrency gate lock"
+    command: echo "--> Start of concurrency gate"
+    concurrency_group: 'validator-bonds/prepare-bidding'
+    concurrency: 1
+
+  - wait: ~
+
+  - label: ":floppy_disk: :arrow_left: :cloud: Downloading Settlements JSON data"
+    commands:
+    - |
+      epoch=${EPOCH:-$(buildkite-agent meta-data get epoch)}
+      buildkite-agent meta-data set epoch "$$epoch"
+    - |
+      curl -s "https://scoring.marinade.finance/api/v1/scores/sam?epoch=$$epoch" -o validators.json
```

Comment:
I believe it would be nice to name the output name differently to validators.json. It's connected to SAM and it's provided to argument --sam-meta-collection so something with SAM. Like sam-scores or at least sam-validators (or rather sam-scores-validators).